sandbox/src/RedirectOffers.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import ContentSecurityPolicy from "./ContentSecurityPolicy";
+
+export default function RedirectOffers() {
+  window
+    .alloy("sendEvent", {
+      renderDecisions: true
+    })
+    .then(({ decisions = [] }) => {
+      console.log("personalized decisions", decisions);
+    });
```

Comment:
This is going to send an event every time this component renders, which probably isn't really what you're after (the component can re-render many times while the user is on the view). If you want to send an event every time the user navigates to this view, you'll probably need to use the `useEffect` React hook. Something like:

```
useEffect(() => {
    window
      .alloy("sendEvent", {
        renderDecisions: true
      })
      .then(({ decisions = [] }) => {
        console.log("personalized decisions", decisions);
      });
  }, []);
  ```